### ISSUE TITLE
Bug - Utilities - In Dark theme, text/background utilities fail contrast check

### ISSUE THREAD

**Issue by srambach** [patternfly/patternfly#4929]
**Describe the problem**
A number of text color and background color utilities fail a11y checks.

**Expected behavior**
Per design: these text colors are not needed in dark mode, so they can be defined as the -100 version.

Text color utilities:
- pf-u-font-color-300
- pf-u-link-color-visited
- .pf-u-default-color-300
- .pf-u-success-color-200
- .pf-u-info-color-200
- .pf-u-danger-color-300

Background colors - Change to the light card background color, since in dark mode, the colored backgrounds are not used in lieu of other markers like border. Note that some of the background/text color issues here may be fixed when the text colors above are fixed.
- pf-u-background-color-default
- pf-u-background-color-success
- pf-u-background-color-info
- pf-u-background-color-warning
- pf-u-background-color-danger
